src/transform/EditTransform.css

```diff
@@ -4,8 +4,9 @@
 }
 
 .pagelib_edit_section_link_container {
-  margin: 2px 8px;
-  float: right;
+  margin: 2px 16px;
+  display: table-cell;
```

Comment:
I would prefer to avoid usage of `display: table-cell` if possible because the rest of the content is not a tabular layout. I see that you've updated the release notes to indicate the problem this solves but I didn't quite follow! Would you mind updating the EditTransform demo with the use case? That would really help with testing and future development!

Here's what I see in the demo before:

![screenshot from 2018-04-06 08-40-07](https://user-images.githubusercontent.com/1071806/38424793-c650f26a-3977-11e8-99d8-c7b057962671.png)

And after, which doesn't quite look right:

![screenshot from 2018-04-06 08-43-40](https://user-images.githubusercontent.com/1071806/38424801-ca4c943c-3977-11e8-8f2c-027197cdef61.png)